third_party/move/move-compiler-v2/tests/file-format-generator/borrow.exp

```diff
@@ -43,44 +43,53 @@ B0:
 	2: Ret
 }
 mut_field(Arg0: &mut S): u64 /* def_idx: 3 */ {
-L1:	loc0: &mut u64
```

Comment:
Note that these changes are a result of:
1. previously, `OPTIMIZE` being `false` but `OPTIMIZE_WAITING_FOR_COMPARE_TESTS` being incorrectly `true`, as these are meant to show unoptimized generated code (the optimized versions are shown in `.opt.exp`
2. now both these experiments are false.